packages/xo-web/src/xo-app/backup-ng/new/new-schedule.js

```diff
@@ -57,11 +54,17 @@ export default [
           value,
         })
       },
-      setCronTimezone: (_, { cronPattern, timezone }) => state => ({
-        ...state,
-        cron: cronPattern,
-        timezone,
-      }),
+      setCronTimezone: ({ setSchedule }, { cronPattern, timezone }) => () => {
+        setSchedule({
+          name: 'cron',
+          value: cronPattern,
+        }).then(() =>
+          setSchedule({
+            name: 'timezone',
+            value: timezone,
+          })
+        )
```

Comment:
Why can't you do both together?

Reply:
It will not work if the update is done at the same time because we override the schedule.

Reply:
To do this i need to let the `setSchedule` support multiple params

Reply:
What about supporting a syntax like this for `setSchedule`?
```js
setSchedule(
  { name: 'foo', value: foo },
  { name: 'bar', value: bar }
)
```
or even like this (breaks the current code):
```js
setSchedule({ foo, bar })
```

Reply:
Ok, I just saw your other comment :) So if you agree with this, I don't mind splitting it in another PR.